### DISCUSSION TITLE
recipes/desserts/pear-tarte-tatin/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#371] [Comments]
# recipes/desserts/pear-tarte-tatin/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/pear-tarte-tatin/

**Comment by nicholaswilde** [comment]
Add tips from source

**Comment by xtluo** [comment]
Thank you for your email.This is  an automatic reply, which means I have received your email and I will check it as soon as possible.Best wishes !Xiaotao Luo
